### ISSUE TITLE
Add option to include or exclude specific IDs

### ISSUE THREAD

**Issue by ithron** [ithron/SOF-Utils#6]
**Is your feature request related to a problem? Please describe.**
When exporting images using `sof-export-images` eg. for annotation training data, only IDs in the training set should be included, or the other way around, the IDs from the test set should be excluded.

**Describe the solution you'd like**
Add an option to `sof-export-images` to include or exclude IDs listed in a file.

**Describe alternatives you've considered**
Manual deletion of unwanted files after the export, but costs more time and disk space.
